src/grass_gis_helpers/cleanup.py

```diff
@@ -86,6 +92,36 @@ def general_cleanup(
                 quiet=True,
                 stderr=nulldev,
             )
+    for rm_s_wr in rm_strds_w_rasters:
+        if f"{rm_s_wr}@{mapset}" in strds:
+            grass.run_command(
+                "t.remove",
+                flags="fd",
+                type="strds",
+                input=rm_s_wr,
+                quiet=True,
+                stderr=nulldev,
+            )
```

Comment:
```suggestion
    for rm_r_wr in rm_strds_w_rasters:
        if f"{rm_r_wr}@{mapset}" in strds:
            grass.run_command(
                "t.remove",
                flags="fd",
                type="strds",
                input=rm_r_wr,
                quiet=True,
                stderr=nulldev,
            )
```
maybe change, to be consistent with vector changes below (same for above `rm_strds`)